common/djangoapps/student/views/dashboard.py

```diff
@@ -598,8 +598,8 @@ def student_dashboard(request):
         'EMPTY_DASHBOARD_MESSAGE', None
     )
 
-    course_limit_in_req = request and 'course_limit' in request.GET
```

Comment:
@adeelehsan you should change the text of the parameter_name as well.  eg. `course_limit` -> `disable_course_limit`